cmd/datamon/cmd/root.go

```diff
@@ -150,15 +136,23 @@ func initConfig() {
 	//  do not require config to be set for all commands
 }
 
-func requireFlags(cmd *cobra.Command, flags ...string) {
-	for _, flag := range flags {
-		err := cmd.MarkFlagRequired(flag)
-		if err != nil {
-			err = cmd.MarkPersistentFlagRequired(flag)
-		}
-		if err != nil {
-			wrapFatalln(fmt.Sprintf("error attempting to mark the required flag %q", flag), err)
-			return
-		}
+func handleConfigErrors(err error) {
```

Comment:
@ransomw1c  hope this partially addresses your concerns ("qualms"...) expressed when reviewing the PR on error handling/wrapping.

Reply:
yes, this uses the declarative [type info] data to make substantive decision about program behavior (as opposed to string formatting), so it's considered the sort of win that adding types beyond `error` could afford that `fmt.Errorf("<new-msg>: %v", cause)` doesn't.

so cf. here from #323 in case the intent (still trying to overload the "concern" buzzword) is not clear from the writeup there.